test/sequential/sequential.status

```diff
@@ -47,8 +47,9 @@ test-watch-mode-inspect: SKIP
 # https://github.com/nodejs/node/issues/41286
 test-performance-eventloopdelay: PASS, FLAKY
 
-[$system==ppc || $system==ppc64]
+[$system==linux && $arch==ppc64]
```

Comment:
re. `$arch` and the confusion in the previous PR's -- it looks like this should match `process.arch`:
https://github.com/nodejs/node/blob/e0b159ee8276c617eb774bafb8e4e1afa5edbf1c/tools/test.py#L1676-L1688
which on ppc64le is `ppc64`:
```console
[iojs@test-osuosl-rhel8-ppc64--le-2 ~]$ ./build/workspace/node-test-commit-plinux/nodes/rhel8-ppc64le/out/Release/node -p process.arch
ppc64
[iojs@test-osuosl-rhel8-ppc64--le-2 ~]$
```